containers/src/Data/Map/Internal.hs

```diff
@@ -1146,7 +1146,8 @@ updateLookupWithKey f0 k0 = toPair . go f0 k0
 
 -- | /O(log n)/. The expression (@'alter' f k map@) alters the value @x@ at @k@, or absence thereof.
 -- 'alter' can be used to insert, delete, or update a value in a 'Map'.
--- In short : @'lookup' k ('alter' f k m) = f ('lookup' k m)@.
+-- In short : @'lookup' k ('alter' f k m) = f ('lookup' k m)@.  Note that
+-- @'adjust' = alter . fmap@.
```

Comment:
I think this paragraph is getting a bit too long. Could you move the example to a new paragraph? And please do the same for the strict version too.

Reply:
Sure. I put the new part under the code block otherwise it looked like the code block was supposed to demonstrate the new part, which it isn't.